CIAOS-Rust/client/ciaos/client.py

```diff
@@ -0,0 +1,183 @@
+# client.py
+
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), 'util'))
+import requests
+from typing import List
+from util.flatbuffer.flatbuffer_handler import create_flatbuffer, parse_flatbuffer
+
+
+def put(user: str, api_url: str, key: str, data_list: List[bytes]):
```

Comment:
Create a config object that the users can use to configure all user details. 
```
config.user.id = ""
config.user.access_key=""
config.api_url = ""

```

So that we don't have to pass in api url and other meta details and make changes to the function everytime